pulp_ansible/tests/functional/api/git/test_sync.py

```diff
@@ -135,3 +135,57 @@ def test_sync_collection_from_git_commit_sha(self):
         self.assertEqual(version.git_url, "https://github.com/ansible-collections/amazon.aws/")
         self.assertEqual(version.git_commit_sha, "d0b54fc082cb63f63d34246c8fe668e19e74777c")
         self.assertEqual(version.download_url, None)
+
+    def test_sync_metadata_only_collection_from_pulp(self):
+        """Sync collections from another Pulp that only has metadata.
+
+        Create a GitRemote
+        Create a repository
+        Sync from the remote
+        Create distribution to serve the repository
+        Create AnsibleRemote pointing to the distribution.
+        Create a second repository.
+        Sync using the AnsibleRemote into the second repository.
+        Assert content is added to the second repository.
+
+        """
+        body = gen_ansible_remote(
```

Comment:
I thought this would be `gen_ansible_git_remote`

Reply:
Oh actually this is fine because this is just generating data that is used further down. Nevermind.